packages/openapi-v3-types/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@loopback/openapi-v3-types",
+  "version": "1.0.0-alpha.1",
```

Comment:
`4.0.0-alpha.1`

Reply:
@virkt25 ah, the version begins with 1 because this module is not tightly related to loopback 4 features, as discussed in https://github.com/strongloop/loopback-next/pull/999#discussion_r168741022

e.g. loopback 3(not implying it would happen) or 5 can still use it.

Reply:
Ah ok. I must've missed the earlier comment. LGTM if this is the approach we want to go with (considering with @raymondfeng 's PR this will become `0.1.0`). This is all the more reason I think version should be locked, otherwise different packages on different versions is going to get confusing in terms of compatibility.

But I think that's a separate discussion. All good otherwise :)

Reply:
If this lands first, make sure you remind @raymondfeng that this will also need to change to 0.1.0 (we can't leave it in the `a.b.c-alpha.x` form if the rest of the monorepo is switching)

Reply:
I changed it to `0.1.0`